retriever/lib/download.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from retriever.engines import choose_engine
+from retriever.lib.tools import name_matches
+from retriever.lib.scripts import SCRIPT_LIST
+
+script_list = SCRIPT_LIST()
+
+
+def download(dataset, path='./', quite=False, subdir=False, use_cache=False, debug=False):
+    """Download scripts for retriever."""
+    args = {
+        'dataset': dataset,
+        'command': 'download',
+        'path': path,
+        'subdir': subdir,
+        'quite': quite,
+        'use_cache': use_cache
+    }
```

Comment:
@ShivamNegi  this code is still replicated in two files. `retriever/lib/download.py` and `retriever/lib/install.py`
Refactor it 
```
 scripts = name_matches(script_list, args['dataset'])
   if scripts:
       for script in scripts:
           print("=> Downloading", script.name)
           try:
               script.download(engine, debug=debug)
               script.engine.final_cleanup()
           except Exception as e:
               print(e)
               if debug:
                   raise
   else:
      .
      .




```